src/ecalib.c

```diff
@@ -46,6 +46,8 @@ int main_ecalib(int argc, char* argv[])
 {
 	long calsize[3] = { 24, 24, 24 }; 
 	int maps = 2;
+	int debug_level_save = debug_level;
+	int debug_level_local = debug_level;
```

Comment:
Could we save and restore 'debug_level' in 'main_bart' ? You are using this is main entry function, or?

Reply:
Done.